lldb/source/Commands/CommandObjectProcess.cpp

```diff
@@ -468,7 +468,23 @@ class CommandObjectProcessContinue : public CommandObjectParsed {
       case 'b':
         m_run_to_bkpt_args.AppendArgument(option_arg);
         m_any_bkpts_specified = true;
-      break;
+        break;
```

Comment:
This isn't how you encode two incompatible options in lldb's command parser.  Just put the two options in two Option Sets and lldb will take care of the rest.  Look for uses of LLDB_OPT_SET for more details.